tests/test_record.py

```diff
@@ -23,17 +24,46 @@ def start_requests(self):
 
     def parse(self, response):
         {parse}
+
+    def parse_item(self):
+        {parse_item}
 '''
 
 
 def run(*pargs, **kwargs):
-    proc = subprocess.Popen(*pargs, **kwargs)
-    proc.wait()
-    return {
-        'returncode': proc.returncode,
-        'stdout': proc.stdout.read(),
-        'stderr': proc.stderr.read(),
-    }
+    try:
+        process = subprocess.run(*pargs, **kwargs)
```

Comment:
I had to switch to subprocess.run since the previous implementation raised lots of warnings about processes not being closed.

Reply:
Thanks for looking into this. I thing this comment is related to the same https://github.com/scrapinghub/scrapy-autounit/pull/25/files/f3e1e95b309347cfccefd9cdcd307f4355acd4e3#r311111880